flytekit/clis/sdk_in_container/pyflyte.py

```diff
@@ -120,6 +122,14 @@ def main(ctx, pkgs: typing.List[str], config: str, verbose: bool):
     if config:
         ctx.obj[CTX_CONFIG_FILE] = config
         cfg = configuration.ConfigFile(config)
+        # Set here so that if someone has Config.auto() in their user code, the config here will get used.
+        if FLYTECTL_CONFIG_ENV_VAR not in os.environ:
+            os.environ[FLYTECTL_CONFIG_ENV_VAR] = config
+        else:
+            cli_logger.warning(
+                f"Config file in arg {config} will be overridden by env var {FLYTECTL_CONFIG_ENV_VAR}:"
```

Comment:
I feel like `args` should take higher precedence than `env`. here is the [doc](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html) for aws credentials precedence.

Reply:
sure.